modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-aws-sts-install_{context}"]
+= Setting up the Amazon S3 storage with the Security Token Service (STS)
```

Comment:
@max-cx could you please note here that this is tech-preview? 

@rubenvp8510 will be working on https://issues.redhat.com/browse/TRACING-4517 and this procedure will change to use CCO operator.

Reply:
ACK. Done (commit amended).